### PR TITLE
Merge branch '3.x' into master

### PR DESCRIPTION
**Pull request by phansys**

### CODE REVIEW

File: src/Admin/AdminExtensionInterface.php [hunk 1]
```diff
@@ -66,7 +66,7 @@ public function configureTabMenu(
     );
 
     /**
-     * @param mixed $object
+     * @param object $object
```
Comment (core23) [inline src/Admin/AdminExtensionInterface.php]: You can remove the doc and use type hints here

File: src/Admin/AdminHelper.php [hunk 2]
```diff
@@ -25,6 +25,8 @@
 use Symfony\Component\Form\FormView;
 
 /**
+ * @final since sonata-project/admin-bundle 3.x
```
Comment (core23) [inline src/Admin/AdminHelper.php]: We don't need this

File: src/Admin/AdminExtensionInterface.php (hunk 1 above)
Comment (core23) [reply src/Admin/AdminExtensionInterface.php]: Maybe we should do the cleanup in another PR.

Comment (phansys) [reply src/Admin/AdminExtensionInterface.php]: I really prefer to do the cleanup in other PR.

**Pull request closed without merging**